--- ./webserver/mongoose.h.orig	2021-01-15 11:36:30.903697386 +0000
+++ ./webserver/mongoose.h	2021-01-15 11:37:02.127899177 +0000
@@ -1762,7 +1762,7 @@ typedef struct {
 
 void cs_md5_init(cs_md5_ctx *c);
 void cs_md5_update(cs_md5_ctx *c, const unsigned char *data, size_t len);
-void cs_md5_final(unsigned char *md, cs_md5_ctx *c);
+void cs_md5_final(unsigned char md[16], cs_md5_ctx *c);
 
 #ifdef __cplusplus
 }
