--- smtube-1.5/src/smtube.pro.qtsingleapplication	2012-11-30 16:46:02.000000000 +0000
+++ smtube-1.5/src/smtube.pro	2012-12-17 03:10:55.424870074 +0000
@@ -5,10 +5,11 @@
 TEMPLATE = app
 QT += network xml
 TARGET = 
-DEPENDPATH += . qtsingleapplication
-INCLUDEPATH += . qtsingleapplication
+DEPENDPATH += .
+INCLUDEPATH += .
 
 CONFIG += qt warn_on release
+CONFIG += qtsingleapplication
 
 RESOURCES = icons.qrc
 
@@ -70,10 +71,6 @@ TRANSLATIONS = translations/smtube_es.ts
                translations/smtube_nn_NO.ts
 
 
-# qtsingleapplication
-SOURCES += qtsingleapplication.cpp qtlocalpeer.cpp
-HEADERS += qtsingleapplication.h qtlocalpeer.h
-
 unix {
     UI_DIR = .ui
     MOC_DIR = .moc
