--- ./src/smplayer.pro.orig	2015-08-03 22:54:00.000000000 +0100
+++ ./src/smplayer.pro	2015-10-02 22:07:32.679161412 +0100
@@ -3,6 +3,7 @@ LANGUAGE = C++
 
 CONFIG += qt warn_on
 CONFIG += release
+CONFIG += qtsingleapplication
 #CONFIG += debug
 
 QT += network xml
@@ -309,14 +310,6 @@ contains( DEFINES, MPLAYER_SUPPORT ) {
 	SOURCES += mplayerprocess.cpp inforeadermplayer.cpp
 }
 
-# qtsingleapplication
-contains( DEFINES, SINGLE_INSTANCE ) {
-	INCLUDEPATH += qtsingleapplication
-	DEPENDPATH += qtsingleapplication
-
-	SOURCES += qtsingleapplication/qtsingleapplication.cpp qtsingleapplication/qtlocalpeer.cpp
-	HEADERS += qtsingleapplication/qtsingleapplication.h qtsingleapplication/qtlocalpeer.h
-}
 
 # Find subtitles dialog
 contains( DEFINES, FIND_SUBTITLES ) {
