--- smplayer-0.8.1/smtube-1.3/src/smtube.pro	2012-05-09 01:12:42.000000000 +0300
+++ smplayer-0.8.1/smtube-1.3/src/smtube.pro	2012-09-20 13:11:52.937317532 +0300
@@ -5,10 +5,11 @@
 TEMPLATE = app
 QT += network xml
 TARGET = 
-DEPENDPATH += . qtsingleapplication
-INCLUDEPATH += . qtsingleapplication
+DEPENDPATH += .
+INCLUDEPATH += .
 
 CONFIG += qt warn_on release
+CONFIG += qtsingleapplication
 
 RESOURCES = icons.qrc
 
@@ -63,10 +64,6 @@
                translations/smtube_hu.ts \
                translations/smtube_pl.ts
 
-# qtsingleapplication
-SOURCES += qtsingleapplication.cpp qtlocalpeer.cpp
-HEADERS += qtsingleapplication.h qtlocalpeer.h
-
 unix {
     UI_DIR = .ui
     MOC_DIR = .moc
--- smplayer-0.8.1/src/smplayer.pro	2012-06-01 18:37:36.000000000 +0300
+++ smplayer-0.8.1/src/smplayer.pro	2012-09-20 13:11:52.937317532 +0300
@@ -3,6 +3,7 @@
 
 CONFIG += qt warn_on
 CONFIG += release
+CONFIG += qtsingleapplication
 #CONFIG += debug
 
 QT += network xml
@@ -235,15 +236,6 @@
         about.ui inputmplayerversion.ui errordialog.ui timedialog.ui \
         toolbareditor.ui
 
-# qtsingleapplication
-contains( DEFINES, SINGLE_INSTANCE ) {
-	INCLUDEPATH += qtsingleapplication
-	DEPENDPATH += qtsingleapplication
-
-	SOURCES += qtsingleapplication.cpp qtlocalpeer.cpp
-	HEADERS += qtsingleapplication.h qtlocalpeer.h
-}
-
 # Find subtitles dialog
 contains( DEFINES, FIND_SUBTITLES ) {
 	DEFINES += DOWNLOAD_SUBS
