--- smplayer-0.8.1/src/smplayer.pro	2012-06-01 18:37:36.000000000 +0300
+++ smplayer-0.8.1/src/smplayer.pro	2012-09-20 13:11:52.937317532 +0300
@@ -3,6 +3,7 @@
 
 CONFIG += qt warn_on
 CONFIG += release
+CONFIG += qtsingleapplication
 #CONFIG += debug
 
 QT += network xml
@@ -235,15 +236,6 @@
         about.ui inputmplayerversion.ui errordialog.ui timedialog.ui \
         toolbareditor.ui
 
-# qtsingleapplication
-contains( DEFINES, SINGLE_INSTANCE ) {
-	INCLUDEPATH += qtsingleapplication
-	DEPENDPATH += qtsingleapplication
-
-	SOURCES += qtsingleapplication.cpp qtlocalpeer.cpp
-	HEADERS += qtsingleapplication.h qtlocalpeer.h
-}
-
 # Find subtitles dialog
 contains( DEFINES, FIND_SUBTITLES ) {
 	DEFINES += DOWNLOAD_SUBS
