diff -rup ./webserver.orig/mongoose.c ./webserver/mongoose.c
--- ./webserver.orig/mongoose.c	2018-05-27 17:59:44.000000000 +0100
+++ ./webserver/mongoose.c	2019-05-19 06:02:13.744763458 +0100
@@ -7821,9 +7821,9 @@ MG_INTERNAL int mg_uri_to_local_path(str
   }
 
 out:
-  LOG(LL_DEBUG,
+/*  LOG(LL_DEBUG,
       ("'%.*s' -> '%s' + '%.*s'", (int) hm->uri.len, hm->uri.p,
-       *local_path ? *local_path : "", (int) remainder->len, remainder->p));
+       *local_path ? *local_path : "", (int) remainder->len, remainder->p)); */
   return ok;
 }
 
