--- ./src/smplayer.pro.qtsingleapplication	2013-05-11 18:48:18.057730407 +0100
+++ ./src/smplayer.pro	2013-05-12 00:39:37.634213140 +0100
@@ -3,6 +3,7 @@ LANGUAGE = C++
 
 CONFIG += qt warn_on
 CONFIG += release
+CONFIG += qtsingleapplication
 #CONFIG += debug
 
 QT += network xml
@@ -263,15 +264,6 @@ FORMS = inputdvddirectory.ui logwindowba
         about.ui inputmplayerversion.ui errordialog.ui timedialog.ui \
         toolbareditor.ui multilineinputdialog.ui
 
-# qtsingleapplication
-contains( DEFINES, SINGLE_INSTANCE ) {
-	INCLUDEPATH += qtsingleapplication
-	DEPENDPATH += qtsingleapplication
-
-	SOURCES += qtsingleapplication.cpp qtlocalpeer.cpp
-	HEADERS += qtsingleapplication.h qtlocalpeer.h
-}
-
 # Find subtitles dialog
 contains( DEFINES, FIND_SUBTITLES ) {
 	DEFINES += DOWNLOAD_SUBS
