--- ./src/smtube.pro.orig	2014-07-24 22:23:22.000000000 +0100
+++ ./src/smtube.pro	2014-09-15 01:38:17.755515996 +0100
@@ -5,7 +5,6 @@
 TEMPLATE = app
 QT += network xml
 TARGET = smtube
-INCLUDEPATH += qtsingleapplication
 
 CONFIG += qt warn_on
 CONFIG += release
@@ -17,7 +16,6 @@ RESOURCES = icons.qrc
 
 DEFINES += NO_SMPLAYER_SUPPORT
 DEFINES += YT_USE_SCRIPT
-DEFINES += USE_SINGLE_APPLICATION
 DEFINES += YT_DL
 
 isEqual(QT_MAJOR_VERSION, 5) {
