--- ./src/smtube.pro.qtsingleapplication	2013-05-03 22:03:39.000000000 +0100
+++ ./src/smtube.pro	2013-05-12 00:47:12.122873411 +0100
@@ -5,10 +5,11 @@
 TEMPLATE = app
 QT += network xml
 TARGET = 
-DEPENDPATH += . qtsingleapplication
-INCLUDEPATH += . qtsingleapplication
+DEPENDPATH += .
+INCLUDEPATH += .
 
 CONFIG += qt warn_on release
+CONFIG += qtsingleapplication
 
 RESOURCES = icons.qrc
 
@@ -93,8 +94,6 @@ TRANSLATIONS = translations/smtube_es.ts
 
 
 # qtsingleapplication
-SOURCES += qtsingleapplication.cpp qtlocalpeer.cpp
-HEADERS += qtsingleapplication.h qtlocalpeer.h
 
 unix {
     UI_DIR = .ui
