diff -rup smplayer-18.2.0/webserver/Makefile smplayer-18.2.2/webserver/Makefile
--- smplayer-18.2.0/webserver/Makefile	2017-01-15 11:13:39.000000000 +0000
+++ smplayer-18.2.2/webserver/Makefile	2018-02-19 17:01:16.489112267 +0000
@@ -2,7 +2,7 @@ PROG = simple_web_server
 MODULE_CFLAGS=-DMG_DISABLE_DAV_AUTH -DMG_ENABLE_HTTP_CGI=0 -DMG_ENABLE_THREADS=0
 
 SOURCES = main.c mongoose.c
-CFLAGS = -O2 -W -Wall -Werror -Wno-unused-function $(CFLAGS_EXTRA) $(MODULE_CFLAGS)
+CFLAGS = -O2 -W -Wall -Werror -Wno-unused-function -Wno-format-truncation -Wno-stringop-truncation $(CFLAGS_EXTRA) $(MODULE_CFLAGS)
 
 ifeq ($(OS), Windows_NT)
 CFLAGS += -lws2_32
